src/OrbitGl/App.cpp

```diff
@@ -993,66 +993,75 @@ void OrbitApp::SendErrorToUi(const std::string& title, const std::string& text)
   });
 }
 
+orbit_base::Future<ErrorMessageOr<std::filesystem::path>> OrbitApp::LoadModuleOnRemote(
+    const std::string& module_file_path) {
+  ScopedStatus scoped_status = CreateScopedStatus(absl::StrFormat(
+      "Searching for symbols on remote instance (module \"%s\")...", module_file_path));
```

Comment:
I see this format is the one that was used in the old implementation, but I suggest moving the module name out of the parenthesis, e.g.:
`"Searching for symbols on remote instance for module \"%s\"..."`
It should also be more consistent with other strings in the status bar related to loading modules.

Reply:
Fixed. Thanks!